File: app/src/main/java/assignment4/exercise5/Main.java
package assignment4.exercise5;

import static assignment4.exercise5.MyKotlinFunctionsKt.add;
import static assignment4.exercise5.MyKotlinFunctionsKt.divide;
import static assignment4.exercise5.MyKotlinFunctionsKt.multiply;
import static assignment4.exercise5.MyKotlinFunctionsKt.subtract;

public class Main {
    public static void main(String[] args) {
        long num1 = 10;
        long num2 = 5;
        long sum = add(num1, num2);
        long difference = subtract(num1, num2);
        long product = multiply(num1, num2);
        long quotient = divide(num1, num2);

        System.out.println("Sum: " + sum);
        System.out.println("Difference: " + difference);
        System.out.println("Product: " + product);
        System.out.println("Quotient: " + quotient);
    }
}

File: app/src/main/java/assignment4/exercise5/Main.kt
package assignment4.exercise5

fun main() {
    val num1: Long = 10
    val num2: Long = 5
    val sum = add(num1, num2)
    val difference = subtract(num1, num2)
    val product = multiply(num1, num2)
    val quotient = divide(num1, num2)

    println("Sum: $sum")
    println("Difference: $difference")
    println("Product: $product")
    println("Quotient: $quotient")
}
